### PR TITLE
Anoosha-GenEpio-01

### PR DESCRIPTION
**Pull request by Anoosha-Sehar**
GRDI Dexa terms curated by Emma and Anoosha

### CODE REVIEW

**Review by cmrn-rhi — APPROVED** [review]

**Comment by cmrn-rhi** [comment]
@ddooley alright with you if we merge this?

**Review by ddooley — CHANGES_REQUESTED** [review]

File: src/ontology/genepio-edit.owl [hunk 1]
```diff
@@ -11471,6 +11487,7 @@ SubClassOf(obo:GENEPIO_0002121 obo:NCIT_C41277)
 
 AnnotationAssertion(obo:IAO_0000114 obo:GENEPIO_0002122 obo:IAO_0000122)
 AnnotationAssertion(obo:IAO_0000117 obo:GENEPIO_0002122 "Emma Griffiths"@en)
+AnnotationAssertion(dc11:contributor obo:GENEPIO_0002122 "A sterilized sampling bag that is puncture proof tabs for protection from damage due to wire-end protrusion and leak-proof closures."@en)
```
[the remark this answers is not in the document]
Comment (Anoosha-Sehar) [reply src/ontology/genepio-edit.owl]: Oh yes, my mistake. Thanks for pointing that out. I just fixed it.

[the remark this answers is not in the document]
Comment (ddooley) [reply src/ontology/genepio-edit.owl]: Ok, but it wasn't just that 'clinical trial sampling strategy'] item I was talking about. There are others sampling strategies that need longer labels too.

[the remark this answers is not in the document]
Comment (Anoosha-Sehar) [reply src/ontology/genepio-edit.owl]: Hmm right. I thought these strategies are usually recognized by these names, like survey study strategy. But maybe it's a better idea to explicitly name them as 'sampling strategy' so to bring more clarity.

[the remark this answers is not in the document]
Comment (ddooley) [reply src/ontology/genepio-edit.owl]: For me the issue is when these terms show up in a search engine and we want to guide people away from making the wrong choice.  The label "field experiment" is a different concept from a "field experiment sampling strategy".

[the remark this answers is not in the document]
Comment (Anoosha-Sehar) [reply src/ontology/genepio-edit.owl]: I agree, and that completely makes sense. Thank you. 
I've updated the labels accordingly. Please let me know if there is any other thing which should be updated.

**Review by ddooley — APPROVED** [review]

**Pull request merged**